### DISCUSSION TITLE
Looking for good first issues to contribute towards

### DISCUSSION THREAD

**Discussion by ZelboK** [microsoft/onnxruntime#19651] [General]
Hi,

I am looking to get started into contributing. The few labels of interest, **contributions welcome, core runtime, and ep:CUDA** are filled with lots of questions from downstream users rather than stories to track. Some are several years old and look to be unprioritized. It is hard to navigate through them all to find a few good issues to start with. 

I have skills with CUDA/GPGPU, C++, & Scala. Could someone point me to a few possible issues?

**Comment by hariharans29** [comment]
Thanks for the offer to contribute! It maye be a good idea to start looking at some of the more recent issues in the repo. Given that you mention CUDA/GPGPU skills, you can start looking at user issues where the issue is about CUDA performance or users have a problem running a model on the CUDA EP (when it runs nicely on some other EP). The latter will give a detailed understanding of the ORT development stack. One recent issue that seems like a good first issue to get started with seems to be - https://github.com/microsoft/onnxruntime/issues/19754.
